solutions/arrays/pascal-triangle.js

```diff
@@ -0,0 +1,37 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+let rowNumber = parseInt(readLine())
+if(rowNumber ===1)
+{
+	console.log(1);
+}
+else
+{
+	let arr=[1]
+	for(let row=2;row<=rowNumber;row++)
+	{
+		let temporaryArray=[]
+		for(let column=1;column<=row;column++)
+		{
+			if(column===1)

```

Comment:
Always use curly braces , even for single line body. Apply this wherever applicable in  this program as well as in your other PRs.